tests/support/sockjs_server.js

```diff
@@ -44,7 +44,7 @@ function startServer(port, config, prefix) {
       res.end(new Array(2049).join('a') + '\nb\n');
     } else if (req.url === '/config.js') {
       if (req.headers.referer) {
-        var parsedOrigin = url.parse(req.headers.referer);
+        var parsedOrigin = new URL(req.headers.referer);
```

Comment:
can you please also check if tests and other things also need adjustment with this change?

Reply:
I ran the tests and they're all passing. The change doesn't impact any functionality as it does the same thing in just more modern *supported* syntax.